modules/library/main/src/main/java/org/geotools/data/Base64.java

```diff
@@ -368,99 +366,6 @@ private Base64() {}
         } // end switch
     } // end encode3to4
 
-    /**
-     * Serializes an object and returns the Base64-encoded version of that serialized object. If the
-     * object cannot be serialized or there is another error, the method will return <tt>null</tt>.
-     * The object is not GZip-compressed before being encoded.
-     *
-     * @param serializableObject The object to encode
-     * @return The Base64-encoded object
-     * @since 1.4
-     */
-    public static String encodeObject(java.io.Serializable serializableObject) {
```

Comment:
So these methods are just being removed? No deprecation cycle ...

Reply:
Nothing was using them, and they are generally dangerous. This class has been incorportated from some other code base, keeping it as is. I can add them back, deprecate, and annotate them so that PMD ignores them.

Reply:
Done as suggested above.